app/models/parser.rb

```diff
@@ -0,0 +1,16 @@
+require 'json'
+
+module Parser
```

Comment:
i like that you have this extracted out :thumbs_up: (idk why my emoji's aren't working!!)